web_common/web_common/templates/header.html

```diff
@@ -22,8 +22,13 @@
 	<div class="batch-caret header-dropdown-menu-caret"></div>
 	{% if userdata['is_developer'] == 1 %}
 	  <a class="header-dropdown-menu-link" href="{{ batch_driver_base_url }}">Driver</a>
-          <a class="header-dropdown-menu-link" href="{{ batch_base_url }}/billing">Billing</a>
 	  <a class="header-dropdown-menu-link" href="{{ batch_base_url }}/billing_projects">Billing Projects</a>
+        {% endif %}
```

Comment:
I think this endif is matched to the is_developer one, so we should align it the same